verl/utils/reward_score/sandbox_fusion/__init__.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+import traceback
+
+from .utils import check_correctness
+
+"""
+Verify code correctness using the sandbox_fusion FaaS service provided by public clouds. Public clouds covered by testing include:
+- volcengine.com
```

Comment:
is it possible to deploy sandbox fusion locally, and does this PR support that kind of usage as well? How do users develop and debug sandbox before onboarding with volc-engine?

Reply:
update this comment